daemon/api_test.go

```diff
@@ -2431,6 +2435,7 @@ func (s *apiSuite) TestInterfaces(c *check.C) {
 
 func (s *apiSuite) TestConnectPlugSuccess(c *check.C) {
 	d := s.daemon(c)
+	s.command = testutil.MockCommand(c, "snap", "")
```

Comment:
It'd be nicer to keep this inline as usual and as a local var:
```
command := tesutil.MockCommand(c, "snap", "")
defer command.Restore()
```
Same for all cases below.